main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"text/template"
+	"os"
+	"os/exec"
+	"time"
+	"go.etcd.io/etcd/clientv3"
+	"context"
+	"path"
+	"github.com/sirupsen/logrus"
+)
+
+type Version struct{
+	PilotVersion string
+	ProdVersion string
+}
+
+func main() {
+	
+	cli, err := clientv3.New(clientv3.Config{
+		Endpoints:   []string{"http://etcd:2379"},
+		DialTimeout: 5 * time.Second,
+	})
+	if err != nil { 
+		logrus.Fatal("Could not connect to etcd")
+		panic(err)
+	}
+	
+	defer cli.Close()
+
+	var versions Version 
+	versionsChan := make(chan Version)
+	
+	go versions.watchUpdatedVersions(cli, versionsChan)
+	
+	for {
+		select{
+		case versions := <-versionsChan:
+			generateAlertFile(versions)
+			updatePrometheus()
+		}
+	}
+	
+}
+
+func generateAlertFile(v Version) {
+	logrus.Info("Generate alert file")
+	
+	tmpl, err := template.ParseFiles("/etc/prometheus/alert-rules.yml.tmpl")
+	if err != nil { 
+		panic(err) 
+	}
+
+	f, err := os.Create("/etc/prometheus/comparative-alerts.yml")
+	if err != nil {
+		panic(err)
+		
+	}
+	err = tmpl.Execute(f, v)
+	if err != nil { 
+		panic(err) 
+	}
+}
+
+func (v Version) watchUpdatedVersions(cli *clientv3.Client, versionsChan chan Version) {
+	watchChan := cli.Watch(context.TODO(), "/versions", clientv3.WithPrefix())
+	
+	for {
+		rspProd, err := cli.Get(context.TODO(), "/versions/"+ os.Getenv("REGISTRY_SERVICE") + "/prod_version", clientv3.WithPrefix())
```

Comment:
@KarineValenca, we should avoid using `os.Getenv`. You endup assuming  some constraints on how the user should be using your utility program. Again, take a look at cobra and viper: by putting them together, we define a clear interface for someone to use out program and we free ourselves from handling multiple types of param input (cli args, os envs and config file).